stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/list/ChannelItem.kt

```diff
@@ -171,7 +175,15 @@ public fun ChannelDetails(
 
         val lastMessageText = channel.getLastMessagePreviewText(currentUser)
 
-        if (lastMessageText.isNotEmpty()) {
+        if (isMuted) {
+            Text(
+                text = "Channel muted",
```

Comment:
I think we have some UI/Design for this so it would be great if we can pull and see if this is what we want (I think this is what iOS does) and then have it translated in resources.

Also - Do we want to show the mute icon for this?

Generally - let's see what the design has in store for us and what we can do with that!